src/app/legacy/containers/PageHandlers/withContexts.jsx

```diff
@@ -60,6 +60,7 @@ const WithContexts = Component => {
             mvtExperiments={mvtExperiments}
             isNextJs={isNextJs}
             isUK={isUK}
+            counterName={atiAnalytics?.pageIdentifier ?? null}
```

Comment:
This is no longer required, and needs to be deleted: see https://github.com/bbc/simorgh/pull/11782/files#r1750005785
```suggestion
```